cmds/exp/netbootxyz/netbootxyz.go

```diff
@@ -175,7 +175,7 @@ func (o OSEndpoint) Load() error {
 			fmt.Printf("With Initrd at %s\n", tmpPath+"initrd")
 			fmt.Printf("Commandline: %s\n", o.Commandline)
 			// Load Kernel and initrd
-			err = kexec.FileLoad(vmlinuz, initrd, o.Commandline)
+			kexec.FileLoad(vmlinuz, initrd, o.Commandline)
 			// Load KExec kernel and initrd - init cmdline
 			cmd := exec.Command("kexec", "-e")
```

Comment:
This is an odd bit of code. We have to be root at this point. so why not just do what kexec does?

                if err := kexec.Reboot(); err != nil {
                        log.Fatalf("%v", err)
it's a bit odd to run a command.

Reply:
I agree, using the kexec pkg is the way to go. see #2539